### DISCUSSION TITLE
v0.61.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#2900] [Announcements]
> Note: This update requires database migrations, take your installation offline, backup the database, and run with yarn db:migrate or yarn db:migrate --env production-ssl-disabled

### Pin to Home

![image](https://user-images.githubusercontent.com/380914/148329822-c3cf968a-2090-4420-9db2-c6224f73c241.png)

This month pinned documents got a visual refresh to help them stand out – we also added the option to reorder pins and pin documents to the home screen for everyone in the team. Make sure to check out all the other enhancements below…

### Enhancements

- Drafts are now displayed in the sidebar while viewing and editing (#2820)
- Added ability to "follow" another user in a document by clicking their avatar (#2858)
- Added "full width" option for documents display (#2869)
- Added recent searches to search screen (#2868) 
- Added '+' button to create new docs from sidebar (#2864)
- Added 'Whimsical' as embed provider
- Added various new collection icons
- It's now possible to replace images in editor
- You can now choose which events publish to Slack (#2857)
- Updated and improved translations
- Views are now recorded for drafts (#2862)
- IAM role authentication now supported for S3 (#2830) 
- 🇻🇳  Vietnamese added as language option

#### Editor

- Fixed emoji text selection behavior
- Document titles in RTL script not correctly aligned
- Github Gist's now load correctly in Safari (#2520)
- Improved inline code display in dark mode (#2835)
- Document hover previews restored when collaborative editing enabled
- Nested list items no longer escape page boundaries
- Emoticons are no longer converted to emojis on save (#2785)

#### Platform

- Fixed incorrect styling of icon color picker in dark mode
- Fixed an issue that would show group creation options to those without permission (#2621)
- Fixed unresponsiveness of select inputs
- Fixed history sidebar behavior
- Fixed duplicate notification for publish event (#2757)
- It's now possible to use email providers with no password (#2767)
- Clickable area for docs in sidebar is no longer constructed (#2809) 
- Removed unused database indexes (#2747)
- Sidebar toggle now displayed on tablets

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.61.0'>v0.61.0</a>.</em>

**Comment by lgg42** [comment]
Amazing! Thank you so much! :pray:

**Comment by max-mykhailenko** [comment]
Thank you, migrated and updated perfectly!

**Comment by nvolker** [comment]
Amazing!
